app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,7 @@
 
   <%= render 'shared/favicons' %>
   <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
why?

Reply:
I was trying to enable jQuery. It was my mistake. Undone.